src/utils/createOptimizedPicture.ts

```diff
@@ -61,7 +61,12 @@ interface CreateOptimizedPictureArgs {
  * @remarks
  * The last breakpoint provided in the `breakpoints` array is used as the source for the fallback image.
  */
-export function createOptimizedPicture(createOptimizedPictureArgs: CreateOptimizedPictureArgs): HTMLPictureElement {
+export function createOptimizedPicture(
+  createOptimizedPictureArgs: CreateOptimizedPictureArgs
+): HTMLPictureElement | undefined {
+  if (createOptimizedPictureArgs.src === '' || createOptimizedPictureArgs.src.startsWith('/default-meta-image.png'))
+    return undefined;
```

Comment:
ich glaube hier hätten wir eher ein "default-meta-image.png" in dem dist folder zu verfügung stellen. Default rendern wir ein default image aber man kann auch sagen dass wir kein image rendern wenn kein image da ist. Lass uns drüber reden wenn du willst....

Reply:
done